cmd/aws-cloud-controller-manager/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apiserver/pkg/server/healthz"
+	"k8s.io/apiserver/pkg/util/logs"
+	"k8s.io/kubernetes/cmd/cloud-controller-manager/app"
+	"k8s.io/kubernetes/cmd/cloud-controller-manager/app/options"
+	_ "k8s.io/kubernetes/pkg/client/metrics/prometheus" // for client metric registration
+	_ "k8s.io/kubernetes/pkg/cloudprovider/providers/aws"
```

Comment:
This is how all out-of-tree providers are doing it now, but in the near future, this import path may change depending on [this proposal](https://docs.google.com/document/d/1YqsPzeG10E59vea5StKFM6W_xZq034eTaf1n649JQvs/edit).

Reply:
Yup, this was just meant to get this repository building a binary, and keep the source of truth in-tree.  Once the shared, generic cloudprovider code moves to staging, these imports will reflect that.  And once the aws code moves into *this repository*, these imports will reflect that too.